src/index.ts

```diff
@@ -5,14 +5,16 @@ import * as globAll from 'glob-all';
 import * as request from 'request';
 import * as fs from 'fs';
 import * as archiver from 'archiver';
+import * as ini from 'ini';
+import * as dateformat from 'dateformat';
 // tslint:disable-next-line
 export var __request: any = request;
 
 /**
  * Copies all of the referenced files to the staging folder
- * @param options 
+ * @param options
  */
-export async function prepublishToStaging(options: RokuDeployOptions) {
+export async function prepublishToStaging(options: RokuDeployOptions): Promise<string> {
```

Comment:
In general, I prefer to let typescript infer the return type. Was there a reason you specified it here?

Reply:
Ah! Was not aware of that. I'm just used to having to do it in Swift. Makes sense why it wasn't provided.